dist/css/main.css

```diff
@@ -0,0 +1,1176 @@
+/* Resetting Styles Start */
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap');
+@import url('https://fonts.googleapis.com/icon?family=Material+Icons');
+@import url('https://fonts.googleapis.com/icon?family=Material+Icons+Outlined');
+@import url('https://fonts.googleapis.com/icon?family=Material+Icons+Round');
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background: #fff;
+  color: #272727;
+  min-height: 100vh;
+  font-family: Lato, 'Helvetica Neue', Helvetica, sans-serif;
+  line-height: 1.4;
+}
+
+a {
+  text-decoration: none;
+  color: #fff;
+}
+
+/*
+  Reseting the size of material UI icons
+*/
+.material-icons.md-18 {
+  font-size: 18px;
+}
+
+.material-icons.md-24 {
+  font-size: 24px;
+}
+
+.material-icons.md-36 {
+  font-size: 36px;
+}
+
+.material-icons.md-48 {
+  font-size: 48px;
+}
+
+.material-icons-outlined.md-18 {
+  font-size: 18px;
+}
+
+.material-icons-outlined.md-24 {
+  font-size: 24px;
+}
+
+.material-icons-outlined.md-36 {
+  font-size: 36px;
+}
+
+.material-icons-outlined.md-48 {
+  font-size: 48px;
+}
+
+/* Resetting Styles End */
+/* Main Content Styles Start */
+.main {
+  text-align: left;
+}
+
+@media screen and (min-width: 768px) {
+  .main {
+    width: 70vw;
+    margin: 0 auto;
+  }
+}
+
+.main-topic {
+  padding: 2.5rem 1rem;
+  border-bottom: 1px solid #d7d7d7;
```

Comment:
Use variables for colors so you don't have to copy-paste the actual value in other places.

Reply:
Hey,
I did use variables only. My SCSS is getting compiled into CSS so it is substituting the variable names with actual values. 
In my code, I've written it as `border-bottom: 1px solid $border-color;`

Reply:
Okay. Clearly, I reviewed all the wrong files :) 
Ignore all irrelevant suggestions.